cajita/src/Cajita_SparseIndexSpace.hpp

```diff
@@ -543,6 +543,12 @@ class SparseMap
       \param key Tile hash key
       \param tile_i, tile_j, tile_k Tile ID in each dimension
     */
+    KOKKOS_INLINE_FUNCTION
```

Comment:
Part of the doxygen issue is that this got inserted with the documentation for the function below - this will need to be moved, then new doxygen added for `value_at`

Reply:
Oh got it! Just updated!